### DISCUSSION TITLE
Több mérőállás leolvasása

### DISCUSSION THREAD

**Discussion by hutibopa** [ZsBT/hass-w1000-portal#19] [Q&A]
Sziasztok!

Nagyon szuper a szkript, napi szinten használom, be van tolva openhab alá. 
A kérdésem az lenne, hogy ha lenne több mérőóra (POD) felhasználási helyem is, akkor hogyan tudnám 
lekérdezni őket, vagyis megkülönböztetni a lekérdezés során?
Az élethelyzet az hogy az ősök óráját is be akarom húzni a felügyeletem alá. 
(először meghatalmazás, majd új POD hozzáadása, de utána hogyan tovább?)
Mert akkor "elvileg" látnám egy payloadban az óraállásokat, de hogy szedném szét őket?

Előre is köszi a válaszotokat.

Üdv,
Tibor

**Comment by ZsBT** [comment]
Ha ugyanaz a portál account, akkor azt hiszem elég, ha más-más néven hozod létre a riportokat. Amennyiben másik accounttal kell belépni, az már karcos mert erre nincs most felkészítve ez az integráció.
